src/content/docs/crowdin/project-management/project-settings/privacy-collaboration.mdx

```diff
@@ -33,7 +38,16 @@ Configure your project privacy settings with the following options:
   </Aside>
 * **Allow offline translation** &ndash; allow translators to download source files to their machines for offline translation and upload translations back into the project. The project owner and managers can always download sources and upload translations, not depending on the option status.
 * **Allow proofreaders to access hidden strings** &ndash; allow proofreaders to work with hidden strings. The project owner and managers can always work with hidden strings, not depending on the option status.
-* **Allow project members to manage glossary terms** &ndash; allow translators and proofreaders to add new glossary terms to the project. The project owner and managers can always add and edit glossary terms, not depending on the option status.
+
+## Glossary Access Settings
```

Comment:
Add the "new" badge to the heading (like here - https://support.crowdin.com/project-settings/#project-language-page-view)

Reply:
It's a nice way to draw the reader's attention to the new feature.

Reply:
Sure, I'll add a badge to this section. To align with the "new feature" badges, I think it would be better to add 
:badge[New]{variant=success}

(like for this article - https://support.crowdin.com/developer/crowdin-apps-module-custom-ai/)

Reply:
Yes, the `success` variant is more suitable here